examples/Kittens/Sample/KittenNode.mm

```diff
@@ -137,16 +137,16 @@ - (ASLayoutSpec *)layoutSpecThatFits:(ASSizeRange)constrainedSize
   _imageNode.preferredFrameSize = _isImageEnlarged ? CGSizeMake(2.0 * kImageSize, 2.0 * kImageSize) : CGSizeMake(kImageSize, kImageSize);
   _textNode.flexShrink = YES;
   
-  return
-  [ASInsetLayoutSpec
-   newWithInsets:UIEdgeInsetsMake(kOuterPadding, kOuterPadding, kOuterPadding, kOuterPadding)
-   child:
-   [ASStackLayoutSpec
-    newWithStyle:{
-      .direction = ASStackLayoutDirectionHorizontal,
-      .spacing = kInnerPadding
-    }
-    children:!_swappedTextAndImage ? @[_imageNode, _textNode] : @[_textNode, _imageNode]]];
+  ASStackLayoutSpec *stackSpec = [[ASStackLayoutSpec alloc] init];
```

Comment:
So cool to get to this point in the patch and see what has been achieved.  It really is a big difference, and is easily one of the most important changes between now and announcing 2.0.